src/components/Thread.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<AppContentDetails id="mail-message">
+	<div id="mail-message">
```

Comment:
the previous component is a `div` as well. Is the `app-content-details` class a problem?

Ref https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/AppContentDetails/AppContentDetails.vue#L24

Reply:
the idea is to remove the AppContentDetails component, and to move all the logic at Home.vue. That was my understanding on how Contact app does. Right now the back arrow is shown currently, it just doesnt work :confounded: